.github/ISSUE_TEMPLATE/join_org.md

```diff
@@ -31,4 +28,4 @@ Please read the [guidelines](https://www.kubeflow.org/docs/about/contributing/#j
 ## Additional Instructions
 - After your PR is merged please wait at least 1 hour for changes to propagate.
 - You will receive an email invite (to your GitHub associated email address) to join Kubeflow on GitHub. Follow the instructions on the email to accept your invitation.
-- If after an hour you haven't received an invite to join the GitHub org (or your invite has expired) please open an issue with an [owner](../../OWNERS) tagged to request follow-up.
+- If after an hour you haven't received an invite to join the GitHub org (or your invite has expired) please open an issue with an [owner](https://github.com/kubeflow/internal-acls/blob/master/OWNERS) tagged to request follow-up.
```

Comment:
These relative paths work when you view the [template file](https://github.com/kubeflow/internal-acls/blob/6236626779f2ec1078d3e2ffeb421643ee8ec696/.github/ISSUE_TEMPLATE/join_org.md) in the repo, but not when you actually create [an issue](https://github.com/kubeflow/internal-acls/issues/832)